tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -169,21 +169,22 @@ const templates = {
       };
   `,
   storybook: {
-    /* eslint-disable @fluentui/max-len */
     main: stripIndents`
       const rootMain = require('../../../.storybook/main');
 
-      module.exports = /** @type {Pick<import('../../../.storybook/main').StorybookConfig,'addons'|'stories'|'webpackFinal'>} */ ({
+      module.exports = /** @type {Omit<import('../../../.storybook/main'), 'typescript'|'babel'>} */ ({
+        ...rootMain,
```

Comment:
now every local sb will use root setup ( this will turn on webpack5 mode as well )